content/en/tracing/troubleshooting/faq_apm.md

```diff
@@ -66,7 +69,9 @@ For example, you might have a single service like `service:test` showing multipl
 - `service:test-mongodb`
 - `service:test-postgresdb`
 
-You can merge the service names using an environment variable like `DD_SERVICE_MAPPING` or `DD_TRACE_SERVICE_MAPPING`, depending on the language. 
+You can use [Inferred Service dependencies (beta)][18]. Inferred external APIs use the default naming scheme `net.peer.name`. For example: `api.stripe.com`, `api.twilio.com`, and `us6.api.mailchimp.com`. Inferred databases use the default naming `scheme db.instance`.
+
+Orm you can merge the service names using an environment variable like `DD_SERVICE_MAPPING` or `DD_TRACE_SERVICE_MAPPING`, depending on the language. 
```

Comment:
```suggestion
Or, you can merge the service names using an environment variable like `DD_SERVICE_MAPPING` or `DD_TRACE_SERVICE_MAPPING`, depending on the language. 
```
Haha, sorry, I made a typo. "Orm" is not a word - I meant to add a comma.

Reply:
Hi Ursula, Thanks for your help on this! Do you need us to add the link on our end?